### ISSUE TITLE
ERROR:  relation "pgagent.pga_job" does not exist

### ISSUE THREAD

**Issue by techtangents** [dbeaver/dbeaver#23298]
### Description

When I use DBeaver against an RDS Postgres 14 database, I occasionally get these errors in the logs:

```
2024-04-01 22:03:41 UTC:172.31.95.222(53878):USER@DB:[8369]:ERROR:  relation "pgagent.pga_job" does not exist at character 15
2024-04-01 22:03:41 UTC:172.31.95.222(53878):USER@DB:[8369]:STATEMENT:  SELECT 1 FROM pgagent.pga_job WHERE 1<>1 LIMIT 1
```

I'm not sure what action is triggering this - I'm just browsing the db tables in the left-hand nav bar and running queries. 

It only started happening recently, so it could be an issue introduced in the latest version.

### DBeaver Version

Community Edition 24.0.1

### Operating System

macOS 14.4.1

### Database and driver

PostgreSQL 14.4 on aarch64-unknown-linux-gnu, compiled by gcc (GCC) 7.3.1 20180712 (Red Hat 7.3.1-6), 64-bit
Postgres client 42.7.2

### Steps to reproduce

Not sure. 

### Additional context

Connecting via Tailscale.

**Comment by LonwoLonwo** [comment]
Hello @techtangents 

This query helps DBeaver understand whether you have pg_jobs in your database or not. Do we need to show them - or not.

Does this error has any impact on your work in DBeaver?

**Comment by techtangents** [comment]
There is no visible impact in DBeaver. However, our sysadmins aren't happy that a db client routinely causes errors on the servers. 

Is there a way to turn this off - e.g. a connection setting?

**Comment by LonwoLonwo** [comment]
Ok, thanks for the report. We removed this behavior. This fix will be available in the next release 24.0.2, on Monday.

**Comment by techtangents** [comment]
Thank you very much. Appreciate the help and the quick resolution.
